pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -183,6 +188,72 @@ func handleErr(reqLogger logr.Logger, status *cv1alpha1.CincinnatiStatus, reason
 	reqLogger.Error(e, reason)
 }
 
+func (r *ReconcileCincinnati) ensureAdditionalTrustedCA(ctx context.Context, reqLogger logr.Logger, instance *cv1alpha1.Cincinnati, resources *kubeResources) error {
+	defaultOpenShiftConfigNS := "openshift-config"
```

Comment:
I think this should be a package level `const` and just `openshiftConfigNamespace`